docs/source/releases.md

```diff
@@ -46,10 +46,10 @@ GA images aren't build from scratch but rather promoted from an existing release
 
 Support matrix table shows the version requirements for a particular **scylla-operator** version. Be sure to match these requirements, otherwise some functionality will not work.
 
-|                    | v1.7        | v1.6        | v1.5        | v1.4        | v1.3        | v1.2        | v1.1        | v1.0       |
-| :----------------: | :---------: | :---------: | :---------: | :---------: | :---------: | :---------: | :---------: | :--------: |
-| Kubernetes         | `>=1.20`    | `>=1.19.10` | `>=1.19.10` | `>=1.19.10` | `>=1.19`    | `>=1.19`    | `>=1.11`    | `>=1.11`   |
-| Scylla OS          | `>=4.3`     | `>=4.3`     | `>=4.3`     | `>=4.3`     | `>=4.2`     | `>=4.2`     | `>=4.0`     | `>=4.0`    |
-| Scylla Enterprise  | `>=2021.1`  | `>=2021.1`  | `>=2021.1`  | `>=2021.1`  | `>=2020.1`  | `>=2020.1`  | `>=2020.1`  | `>=2020.1` |
-| Scylla Manager     | `>=2.2`     | `>=2.2`     | `>=2.2`     | `>=2.2`     | `>=2.2`     | `>=2.2`     | `>=2.2`     | `>=2.2`    |
-| Scylla Monitoring  | `>=3.0`     | `>=3.0`     | `>=1.0`     | `>=1.0`     | `>=1.0`     | `>=1.0`     | `>=1.0`     | `>=1.0`    |
+|                   |    v1.8    |    v1.7    |    v1.6     |    v1.5     |    v1.4     |    v1.3    |    v1.2    |    v1.1    |    v1.0    |
+|:-----------------:|:----------:|:----------:|:-----------:|:-----------:|:-----------:|:----------:|:----------:|:----------:|:----------:|
+|    Kubernetes     |  `>=1.21`  |  `>=1.20`  | `>=1.19.10` | `>=1.19.10` | `>=1.19.10` |  `>=1.19`  |  `>=1.19`  |  `>=1.11`  |  `>=1.11`  |
+|     Scylla OS     |  `>=4.3`   |  `>=4.3`   |   `>=4.3`   |   `>=4.3`   |   `>=4.3`   |  `>=4.2`   |  `>=4.2`   |  `>=4.0`   |  `>=4.0`   |
```

Comment:
we should bump the other releases too, for scylla opensource we support only the 2 latest version (5.0 and 4.6)